Storage/Service/AbstractUrlStorage.php

```diff
@@ -204,6 +204,13 @@ public function addChunk(string $hash, string $chunk, ?string $context = null):
      */
     public function finalizeUpload(string $hash, ?string $context = null): bool
     {
-        return rename($this->getPath($hash, $context, false), $this->getPath($hash, $context));
+        $source = $this->getPath($hash, $context, false);
+        $destination  = $this->getPath($hash, $context);
+        try {
+            \rename($source, $destination);
+        } catch (\Throwable $e) {
+            \copy($source, $destination);
+        }
+        return true;
```

Comment:
should we not return false if copy fails?
Now we always return true unless an error is thrown, but than the return true is not really usefull.

Reply:
Agree, I'll fix it